src/plugins/kibana_react/public/notifications/create_notifications.tsx

```diff
@@ -34,8 +36,8 @@ export const createNotifications = (services: KibanaServices): KibanaReactNotifi
       throw new TypeError('Could not show notification as notifications service is not available.');
     }
     services.notifications!.toasts.add({
-      title,
-      text: <>{body || null}</>,
+      title: isString(title) ? title : reactMount(title),
+      text: isString(body) ? body : reactMount(<>{body || null}</>),
```

Comment:
Would work without this change, but this avoid using a MountWrapper when using actual string.

Reply:
Maybe for simplicitly we can use `reactMount` for any case? If we don't, can we use standard type checking, like `typeof title === 'string'` instead importing something from `lodash`.

Reply:
Agreed, the helper function should be able to handle this just fine.

Reply:
Ok, will remove the useless check then.